lib/ansible/plugins/connection/winrm.py

```diff
@@ -275,8 +275,8 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
         except Exception:
             traceback.print_exc()
             raise AnsibleConnectionFailure("failed to exec cmd %s" % cmd)
-        result.std_out = to_bytes(result.std_out)
-        result.std_err = to_bytes(result.std_err)
+        result.std_out = result.std_out
+        result.std_err = result.std_err
```

Comment:
I remember having to add the to_bytes()... it could be that an old version of winrm returned text here.  I do not remember for sure if that was fixed in the lowest winrm version that we support.

Two other notes:
1) This needs to return bytes on python3 as well as python2
2) to_bytes() is a no-op (overhead of the function call and a couple conditionals but no encoding is done) if the string is already a byte string.